freedom-ldscript-generator.c++

```diff
@@ -255,9 +266,15 @@ static void dts_memory (void)
                 });
         });
 
-    if (testram_count > 0)
-        alias_memory("testram", "ram");
-    else if (memory_count > 0) {
+    if (testram_count > 0) {
+    	if (sram_count > 0) {
+            alias_memory("sram0", "ram");
+            alias_memory("sram1", "itim");
```

Comment:
What happens when sram0 is itim and sram1 is ram? How long until we have to support a device with that mapping?

Reply:
It is arbitrary for now